recipes/godot-cpp/config.yml

```diff
@@ -5,3 +5,5 @@ versions:
     folder: 3.x.x
   "cci.3.2-20200130":
     folder: 3.x.x
+  "4.1.1":
+    folder: all
```

Comment:
I would name the folder `4.x.x`

`all` only makes sense if there are no further directories

Reply:
I agree. Would also make the PR diff cleaner.